### DISCUSSION TITLE
How to create detection annotation?

### DISCUSSION THREAD

**Discussion by VishwaBhushan** [rafaelpadilla/review_object_detection_metrics#59] [Q&A]
Hello,

I was wondering if there is any tool / repo for creating detection annotations for tensorflow/yolo/tensorRT/deepstream models. Mainly tensorflow and yolo models.

Any pointers will be appreciated.

Thanks !!

**Comment by rafaelpadilla** [comment]
There are labeling tools that can output bounding boxes into different formats.

I have worked with [CVAT](https://github.com/openvinotoolkit/cvat) and it outputs bounding boxes in different formats:
CVAT, Datumaro, LabelMe, MOT, MOTS, MS COCO, Pascal VOC, YOLO, TFRecord, ImageNet, Wider Face, CamVid, VGGFace2, Market-1501, ICDAR13/15

I never needed to include other formats, but according to [their documentation](https://openvinotoolkit.github.io/cvat/docs/for-users/formats/), they can support customized formats.

This is the only tool that I have used to annotate images and videos.

**Reply by VishwaBhushan** [reply]
Hey @rafaelpadilla thanks for the quick reply. I think I didn't clear myself at first. These tools which you mentioned would be to create ground truth annotations. My question was about how to create the "detection" dataset. Like when I have trained my model (yolo or ssd or deepstream etc) I want to pass a set of images to it and it should create the "detection" dataset in the required format so that I can compare it with the ground truth dataset using your tool.

Thanks
